community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java

```diff
@@ -528,7 +528,7 @@ public static void doImport( PrintStream out, PrintStream err, File storeDir, Fi
 
     private static Config loadDbConfig( File file ) throws IOException
     {
-        return file != null && file.exists() ? new Config( MapUtil.load( file ) ) : Config.defaults();
+        return file != null && file.exists() ? Config.embeddedDefaults( MapUtil.load( file ) ) : Config.defaults();
```

Comment:
Would be nice but that would introduce a cyclic dependency since ConfigLoader is in dbms, and dbms already depends on importtool.